Gruntfile.js

```diff
@@ -44,13 +44,26 @@ module.exports = function (grunt) {
                     }
                 }
             }
+        },
+
+        shell: {
+            health_check: {
+                command: 'mkdir -p artifacts/test/health-check &&' +
```

Comment:
For local testing we don't need to create result files. Can we remove this and simply declare a new script in package.json?